AudioTranscriber.py

```diff
@@ -82,7 +186,11 @@ def process_speaker_data(self, data):
         return temp_file
 
     def get_transcription(self, file_path):
-        result = self.audio_model.transcribe(file_path, fp16=torch.cuda.is_available())
+        if self.lang == "en":
```

Comment:
We shouldn't have to call load_model every time we call get_transcription the model should only be loaded when we initialize or change the language